### PR TITLE
Updated for a account close example

### PR DESCRIPTION
**Pull request by gked**
Fixing  #226
The example showing how to close an account was incorrect. Fixed it by adding a required argument `--sat_per_vbyte`

### CODE REVIEW

File: docs/accounts.md [hunk 1]
```diff
@@ -110,6 +110,5 @@ Incrementally _withdrawing_ from an account is also supported. The command is si
 Finally, if you wish to send _all_ your funds elsewhere, it's possible to close your account out before the main expiration period. We can close out the account we created above with the following command:
 
 ```text
-🏔 pool accounts close --trader_key=0288096be9917f8ebdfc6eb2701635fe658f4eae1e0274dcce41418b3fb5145732
+🏔 pool accounts close --trader_key=0288096be9917f8ebdfc6eb2701635fe658f4eae1e0274dcce41418b3fb5145732 --sat_per_vbyte 0
```
Comment (guggero) [inline docs/accounts.md:113]: nit: maybe use another value than 0 as that will not work? Perhaps 10 or something like that (not too low as that wouldn't work with the current mempool).

**Review by guggero — CHANGES_REQUESTED** [review]
Thanks for the PR!

Just need a different value than 0 then we're good to merge this.

File: docs/accounts.md (hunk 1 above)
Comment (gked) [reply docs/accounts.md:113]: sure thing. I can update to the value I used when closing my account few days ago.

**Review by halseth — CHANGES_REQUESTED** [review]
Thanls for the PR! Can you squash the commits down to one? Otherwise this LGTM 🤠

**Comment by guggero** [comment]
I'll "merge and squash" instead to get this in.

**Review by guggero — APPROVED** [review]

**Pull request merged**